src/common/services/auth.js

```diff
@@ -0,0 +1,108 @@
+/*
+** This service is used to authenticate users 
+** author: @ishmeet-kuliza
+*/
+angular.module('Vyomo')
+.factory('auth', ['$http', '$q', 'env', function($http, $q, env) {
+
+  var userObj,
```

Comment:
can we initialize userObj with initial values, or make a user class and make its objs later when required ?